docs/common-issues/index.md

```diff
@@ -35,7 +35,22 @@ require_once(ABSPATH . 'wp-settings.php');
 
 remains in the `wp-config.php` file, so if you've modified or moved it, put it back there. It gets matched by a regex when WP-CLI runs.
 
-See also: [#1631](https://github.com/wp-cli/wp-cli/issues/1631)
+### PHP Fatal error: Call to undefined function \<WordPress function\>
+
+WP-CLI uses a [custom version](https://github.com/wp-cli/wp-cli/blob/master/php/wp-settings-cli.php) of WordPress's `wp-settings.php` file. Before WP-CLI can load `wp-settings-cli.php`, it needs to know all of the constants defined in `wp-config.php` (database connection details and so on). Because WP-CLI doesn't want WordPress to load yet when it's [pulling the constants](https://github.com/wp-cli/wp-cli/blob/master/php/wp-cli.php#L22) out of `wp-config.php`, [it uses regex](https://github.com/wp-cli/wp-cli/blob/master/php/WP_CLI/Runner.php#L324) to strip the `require_once(ABSPATH . 'wp-settings.php');` statement.
```

Comment:
> WP-CLI uses a [custom version](https://github.com/wp-cli/wp-cli/blob/master/php/wp-settings-cli.php) of WordPress's `wp-settings.php` file.

This is incorrect; the behavior was changed to load `wp-settings.php` in WP 4.6 and higher. You can drop this sentence.

> Before WP-CLI can load `wp-settings-cli.php`, it needs to know all of the constants defined in `wp-config.php` (database connection details and so on).

I'd change the `wp-settings-cli.php` reference to `wp-settings.php`, then it will be correct.